src/App/CabanasRD/Extensions/TaskExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace CabanasRD.Extensions
+{
+    public static class TaskExtensions
+    {
+        public static async void Await(this Task task, Action completedCallback, Action<Exception> errorCallBack)
```

Comment:
Loved it! Veo que sigues a Brian Lagunas jeje

Reply:
Yes Sir!